data/tlg0062/tlg022/tlg0062.tlg022.perseus-eng2.xml

```diff
@@ -32,17 +32,23 @@
 <title>Lucian</title>
 <editor>
 <persName>
-<name>Austin Morris Harmon</name>
+<name role="translator">Austin Morris Harmon</name>
 </persName>
 </editor>
 <author ref="urn:cts:greekLit:tlg0062">Lucian of Samosata</author>
 <imprint>
-<publisher>Trustees of Tufts University</publisher>
-<pubPlace>Cambridge, MA</pubPlace>
+ <pubPlace>London</pubPlace>
+ <publisher>William Heinemann Ltd.</publisher>
+ <pubPlace>Cambridge, MA</pubPlace>
+ <publisher>Harvard University Press</publisher>
+ <date>1915</date>
```

Comment:
two dates?

Reply:
I may have just copied it in twice by accident. Thanks!